### ISSUE TITLE
Bug with Butch in Hub

### ISSUE THREAD

**Issue by KeyboardGecko** [rotators/Fo1in2#95]
In Far Go Traders, you can take the quest about attacks on caravans. Then talk to Butch again to report it was mutants, not lifting a finger to figure it out.

**Comment by Lexx2k** [comment]
Did you "read" the holodisk / bring it as proof?

**Comment by KeyboardGecko** [comment]
I did nothing, just talked to Butch 2 times in a row. Took the quest, reported about Mutants.

**Comment by Lexx2k** [comment]
Did you visit the military base already?

**Comment by KeyboardGecko** [comment]
Nope. Vault 15, Shady sands, Junktown, The Hub. That's it.

**Comment by KeyboardGecko** [comment]
https://disk.yandex.ru/d/cVu7lB9BhElTbQ - the save.
Talk to Butch, talk to Rutger, talk to Butch, agree for the job, talk about the deathclaw.
Talk to Butch again - here it is.

**Comment by KeyboardGecko** [comment]
I made a little video for you.
https://disk.yandex.ru/i/L6cY2t6cNt2GcA

**Comment by Lexx2k** [comment]
Yeah, can't reproduce. I went through the dialog in exactly the same way as you did and the dialog option doesn't show up. Tried it in your savegame, made a new character, booted up latest repo version and tried it again - everywhere the same result.

**Comment by KeyboardGecko** [comment]
1.5, straight from this git, downloaded 2 days ago.
